schema.sql

```diff
@@ -0,0 +1,10 @@
+/* Database schema to keep the structure of entire database. */
+
+CREATE TABLE vet_clinic (
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_create_tables.md#:~:text=named%20vet_clinic.-,Create%20a%20table%20named%20animals%20with%20the%20following%20columns,-%3A), please name this table `animals` instead of `vet_clinic` as you did and don't forget to edit your queries according to that change.